app/assets/javascripts/service_details.js.coffee

```diff
@@ -58,3 +58,5 @@ $(document).ready ->
 
   $('#arms-table').on 'all.bs.table', ->
     $('.name-validation').tooltip()
+    $('.subject-count').tooltip()
```

Comment:
Probably doesn't make a huge difference, but usually we just do $("[data-toggle='tooltip']").tooltip()